thisweek/_posts/2021-12-27-changelog-109.adoc

```diff
@@ -0,0 +1,58 @@
+= Changelog #109
+:sectanchors:
+:page-layout: post
+
+Commit: commit:68319187d63707fa36d7c215ed0e444e87d9652a[] +
+Release: release:2021-12-27[]
+
+== Sponsors
+
+**Become a sponsor:** On https://opencollective.com/rust-analyzer/[OpenCollective] or
+https://github.com/sponsors/rust-analyzer[GitHub Sponsors].
+
+== Known Issues
+
+There are some recent known issues related to the VS Code extension and the update mechanism:
+
+* Since last week, the stable extension contains the server binary, avoiding the need to download it on the first load. Unfortunately, Open VSX, which is the marketplace used by VSCodium and other free Code builds https://github.com/rust-analyzer/rust-analyzer/issues/11080[does not] support this feature, sometimes serving users binaries for the wrong platform. The workaround is to install the non-free version of Code, or to manually download the VSIX for your platform from GitHub releases.
```

Comment:
It might be useful to rephrase this a bit: “if you use X, than do Y. This is caused by Z”. That is, emphasize “am I affected” and “what should I do”, rather than the underlying issue.